<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDiagnosisDialog</class>
 <widget class="QDialog" name="AddDiagnosisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Add Diagnosis</string>
   </property>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="descriptionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="descriptionEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="revertButton">
      <property name="text">
       <string>Revert</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
